articles/confidential-ledger/manage-aad-token-based-users.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Managed AAD Token Based Users in Azure Confidential Ledger
+description: Learn how to manage AAD token based users in Azure Confidential Ledger
+author: settiy
+ms.author: settiy
+ms.date: 02/09/2023
+ms.service: confidential-ledger
+ms.topic: how-to
+---
+
+# Manage AAD Token Based Users in Azure Confidential Ledger
+
+AAD-based users are identified by their AAD object id.
+
+Users with Administrator privileges can manage users of the Confidential Ledger. Available roles are Reader (read-only), Contributor (read and write), and Administrator (read, write, and manage users).
+
+The following client libraries can be used to manage users. Learn more about each by following the links below:
+
+- Python
+- .NET
+- Java
+
+> [!NOTE]
+> Login to Azure using either the Azure CLI or Azure PowerShell. The instructions are available at https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli or https://learn.microsoft.com/en-us/powershell/azure/authenticate-azureps?view=azps-8.3.0 respectively.  
+
+Get the Confidential Ledger's id and the identity service endpoint from the Azure Portal as it is needed to create a client to manage the users. This image shows the properties in the Azure portal.
+
+:::image type="content" source="./media/ledger-properties.png" alt-text="Ledger Properties":::
```

Comment:
I am not able to see the image, maybe I am not viewing it correctly. I usually follow this method to [display images with markdown ](https://marinegeo.github.io/2018-08-10-adding-images-markdown/#:~:text=Images%20can%20be%20added%20to%20any%20markdown%20page,mouseover%22%29.%20%21%20%5BMarineGEO%20circle%20logo%5D%20%28%2Fassets%2Fimg%2FMarineGEO_logo.png%20%22MarineGEO%20logo%22%29). It may be a viewing error on my end though.

Reply:
I followed the format in the other mark down pages.